.github/CODEOWNERS

```
@@ -14,7 +14,8 @@
 convert2rhel/             @oamg/convert2rhel-developers
 
 # Integration tests
-tests/integration         @danmyway @kokesak @SerCantus
+tests/integration         @oamg/conversions-qe
```

```suggestion
tests/         @oamg/conversions-qe
```
Please include all subdirectories.

---

done

---

Great service. 10/10 would recommend